library/private/interactor_impl.h

```diff
@@ -35,15 +35,20 @@ class interactor_impl : public interactor
   interactor_impl(options& options, window_impl& window, scene_impl& scene);
   ~interactor_impl() override;
 
-  interactor& addCommandCallback(
-    std::string action, std::function<bool(const std::vector<std::string>&)> callback) override;
-  interactor& removeCommandCallback(const std::string& action) override;
+  interactor& initializeDefaultCommands() override;
```

Comment:
This one is still a bit long. Can we come up with a shorter meaningful name? Maybe `resetCommands`/`resetBindings`? Or just `initCommands`/`initBindings`?

Reply:
initCommands/initBindings works for me